<!DOCTYPE HTML>
<!--
	Arcana by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>X.X. Maurer-Alcalá</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body class="is-preload">
		<div id="page-wrapper">

			<!-- Nav -->
			<center>
				<nav id="nav">
					<ul>
						<li class="current"><a href="index.html">Research</a></li>
						<!--li><a href="research.html">Research</a></li-->
						<li><a href="about-me.html">About</a></li>
						<li><a href="publications.html">Publications</a></li>
						<li><a href="https://github.com/xxmalcala">GitHub</a></li>
						<li><a href="publication_pdfs/XXMA_CV_10_2023.pdf">Curriculum-Vitae</a></li>
					</ul>
				</nav>
			</center>

			<!-- Gigantic Heading -->
				<section class="wrapper style2">
					<div class="container">
						<header class="major">
							<h2>Eukaryotic Genome Evolution and Biodiversity</h2>
							<p>Bridging 2+ billion years to explore the evolution of eukaryotes,
								their genomes,<br>and diversification through the lense of microbial eukaryotes</p>
						</header>
					</div>
				</section>

				<!-- Main -->
					<section class="wrapper style1">
						<div class="container">
							<div id="content">

								<!-- Content -->

									<article>
										<span class="image featured"><img src="images/banner.jpg" alt="" /></span>

										<h3>Placing Eukaryotes in Context</h3>
										<p>Constructing the eukaryotic tree of life (eToL) presents a unique
											on-going challenge as new deeply diverging lineages are described each year.
											While keeping pace with emergent ancient lineages is challenging,
											construction of a robust eToL is critically important for studying
											the evolution of eukaryotes and their subsequent diversification.
											My work aims to reconcile multiple approaches to infer
											robust relationships at challenging nodes in the eToL (e.g., the
											monophyly of "Archaeplastida" and "Excavata") as well as "orphan"
											lineages that likely have importance regarding the "root" of the eukaryotic
											tree of life (particularly Malawimonads). This work includes
											incorporating multiple phylogenomic techniques and forms of evidence,
											aiming to generate robust hypotheses to explore eukaryotic evolution
											from the last common ancestor (~1.8 billion years ago) to now!
										<br><br>
										&nbsp &nbsp <strong>Relevant publications</strong>
										<br>
											&nbsp &nbsp &nbsp <a href="publication_pdfs/2016_Tekle_etal_Phylogenomics_Discosea_MBE.pdf">
											Tekle <i>et al.</i>, 2016</a><br>
											&nbsp &nbsp &nbsp <a href="publication_pdfs/2022_Sierra_etal_Foram_Phylogeny_MPE.pdf">
												Sierra <i>et al.</i> 2022</a><br>
											&nbsp &nbsp &nbsp <a href="publication_pdfs/2022_Ma_etal_Karyo_Phylogeny_MPE.pdf">
												Ma <i>et al.</i> 2022</a><br>
										</p>

										<h3>How does genetic novelty arise and at what pace?</h3>
										<p>Phylogenomics provides a criticial means to explore the evolution
											and subsequent diversification of genetic novelty at many scales,
											from sibling species, phyla, 'supergroups' and ultimately to major domains of life.
											Building upon PhyloToL, my work has aimed to develop approaches and tools
											to aid in understanding the evolution of young genes and gene families
											from predominantly single-cell 'omics data through phylogenomic and
											machine learning approaches. This avenue of work is making it
											feasible to explore the evolution of genetic novelty, from
											<i>de novo</i> genes to inferring the evolutionary pacing
											of gene family birth/death and expansion/contract over deep time
											scales. With an emphasis on lineages of eukaryotes where single-cell
											'omics approaches dominate the available pools of data (e.g., ciliates
											and amoebozoans), there is ample opportunity to understand the evolutionary
											biology of genetic 'novelty' across eukaryotes and its contributions to the
											on-going diversification of major eukaryotic lineages.
										<br><br>
										&nbsp &nbsp <strong>Relevant publications</strong>
										<br>
											&nbsp &nbsp &nbsp Maurer-Alcalá <i>et al., In Press.</i><br>
										</p>


										<h3>Linking Genome Biology to Genome Evolution</h3>
										<p>When considering genome evolution, we often overlook the impacts of
											organismal biology and life history. Using ciliates, my work has focused
											on how their unusual genome biology (including developmental rearrangements and
											somatic hyperpolyploidy) manifest in their genome evolution.
											While ciliates represent extremes, their genome and developmental
											biology manifest in their evolution in interesting ways; extreme genome
											processing is tied to relaxed evolutionary pressures (Yan,
											<strong>Maurer-Alcalá</strong>, <i>et al.</i>, 2019) and may
											impact gene family birth/death too (<strong>Maurer-Alcalá</strong>,
											<i>et al.</i>, 2018, <strong>Maurer-Alcalá</strong>, <i>et al.</i>, In Press).
											This work has bridged diverse areas of biology (development, evolution,
											bioinformatics) and I am aiming to further explore these questions using
											experimental evolution and molecular biology techniques.
											<br><br>
											&nbsp &nbsp <strong>Relevant publications</strong>
											<br>
												&nbsp &nbsp	&nbsp <a href="publication_pdfs/2016_MaurerAlcala_etal_NucArch_Chilo_GBE.pdf">
													<strong>Maurer-Alcalá</strong> and Katz, 2016</a><br>
												&nbsp &nbsp	&nbsp <a href="publication_pdfs/2018_MaurerAlcala_etal_SCG_Chilo_mBio.pdf">
													<strong>Maurer-Alcalá</strong>, <i>et al.</i>, 2018</a><br>
												&nbsp &nbsp	&nbsp <a href="publication_pdfs/2019_Yan_etal_GenArch_Evo_mBio.pdf">
													Yan, <strong>Maurer-Alcalá</strong>, <i>et al.</i>, 2019</a><br>

										<h3>Developing Bioinformatic Approaches and Tools</h3>
										<p>Phylogenomics and comparative 'omics often use increasingly large datasets
											that present unique computational challenges. An important aspect
											of my work involves	the design and development of computational tools
											and approaches that improve access to those with less computational
											expertise. This includes <a href=https://github.com/Katzlab/PhyloTOL>PhyloToL</a>,
											a phylogenomic pipeline with wide applications , tools for <a href=https://github.com/xxmalcala/DSaGA>
											comparing genomes</a> in ciliates (crazy microeukaryotes)
											to <a href=https://github.com/xxmalcala/TIdeS>machine learning tools
											</a> to aid in data processing and phylogenomics!
											<br><br>
											&nbsp &nbsp <strong>Relevant publications</strong><br>
											&nbsp &nbsp	&nbsp Phylotol:<br>
													&nbsp &nbsp	&nbsp &nbsp <a href="publication_pdfs/2019_Ceron_Romero_etal_PhyloToL_MBE.pdf">
													Ceron-Romero <i> et al.</i>, 2019</a><br>
													&nbsp &nbsp	&nbsp &nbsp	<a href="publication_pdfs/2022_Sierra_etal_Foram_Phylogeny_MPE.pdf">
														Sierra <i> et al.</i>, 2022</a><br>
												&nbsp &nbsp	&nbsp Ciliate Germ-Soma:<br>
													&nbsp &nbsp	&nbsp &nbsp <a href="publication_pdfs/2018_MaurerAlcala_etal_SCGT_GBE.pdf">
														<strong>Maurer-Alcalá</strong>, Yan, <i>et al.</i>, 2018</a><br>
													&nbsp &nbsp	&nbsp &nbsp <a href="publication_pdfs/2018_MaurerAlcala_etal_SCG_Chilo_mBio.pdf">
														<strong>Maurer-Alcalá</strong>, <i>et al.</i>, 2018</a><br>
													&nbsp &nbsp	&nbsp &nbsp <a href="publication_pdfs/2020_Smith_etal_Schmid_Genome_GBE.pdf">
														Smith <i> et al.</i> 2020</a><br>
										</p>
									</article>

							</div>
						</div>
					</section>

				<!-- Footer -->
					<div id="footer">

						<!-- Copyright -->
							<div class="copyright">
								<ul class="menu">
									<li>&copy; Xyrus X. Maurer-Alcalá</li><!--li>Design: <a href="http://html5up.net">HTML5 UP</a></li-->
								</ul>
							</div>

					</div>

			</div>

			<!-- Scripts -->
				<script src="assets/js/jquery.min.js"></script>
				<script src="assets/js/jquery.dropotron.min.js"></script>
				<script src="assets/js/browser.min.js"></script>
				<script src="assets/js/breakpoints.min.js"></script>
				<script src="assets/js/util.js"></script>
				<script src="assets/js/main.js"></script>

		</body>
	</html>
