<!DOCTYPE HTML>
<!--
	Arcana by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>X.X. Maurer-Alcalá</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body class="is-preload">
		<div id="page-wrapper">

				<!-- Nav -->
				<center>
					<nav id="nav">
						<ul>
							<li class="current"><a href="research.html">Research</a></li>
							<li><a href="publications.html">Publications</a></li>
							<li><a href="https://github.com/xxmalcala">GitHub</a></li>
							<li><a href="publication_pdfs/XXMA_CV_11_2024.pdf">Curriculum-Vitae</a></li>
						</ul>
					</nav>
				</center>

			<!-- Main -->
				<section class="wrapper style1">
					<div class="container">
						<div id="content">

							<!-- Content -->

								<article>
									<header>
										<h2>Evolution of Eukaryotes and Their Diversification</h2>
										<p>Bridging 2+ billion years of evolution to explore our origins</p>
									</header>

									<!-- <span class="image featured"><img src="images/banner.jpg" alt="" /></span> -->

									<h3>Placing Eukaryotes in Context</h3>
									<p>Constructing the eukaryotic tree of life (eToL) presents a unique
										on-going challenge as new deeply diverging lineages are described each year.
										While keeping pace with emergent ancient lineages is challenging,
										construction of a robust eToL is critically important for studying
										the evolution of eukaryotes and their subsequent diversification.
										My work aims to reconcile multiple approaches to infer
										robust relationships at challenging nodes in the eToL (e.g., the
										monophyly of "Archaeplastida" and "Excavata") as well as "orphan"
										lineages that likely have importance regarding the "root" of the eukaryotic
										tree of life (particularly Malawimonads). This work includes
										incorporating multiple phylogenomic techniques and forms of evidence,
										aiming to generate robust hypotheses to explore eukaryotic evolution
										from the last common ancestor (~1.8 billion years ago) to now!
									<br><br>
									&nbsp &nbsp <strong>Relevant publications</strong>
									<br>
										&nbsp &nbsp &nbsp <a href="publication_pdfs/2022_Sierra_etal_Foram_Phylogeny_MPE.pdf">
											Sierra <i>et al.</i>, 2022</a><br>
										&nbsp &nbsp &nbsp <a href="publication_pdfs/2022_Ma_etal_Karyo_Phylogeny_MPE.pdf">
											Ma <i>et al.</i>, 2022</a><br>
										&nbsp &nbsp &nbsp <a href="publication_pdfs/2016_Tekle_etal_Phylogenomics_Discosea_MBE.pdf">
											Tekle <i>et al.</i>, 2016</a><br>
									</p>

									<h3>How does genetic novelty arise and at what pace?</h3>
									<p>Phylogenomics provides a criticial means to explore the evolution
										and subsequent diversification of genetic novelty at many scales,
										from sibling species, phyla, 'supergroups' and ultimately to major domains of life.
										Building upon PhyloToL, I am developing approaches and tools (e.g., TIdeS, PhyG)
										to aid in understanding the evolution of these "innovations" (e.g.,
										lateral gene transfers, <i>de novo</i> and lineage-specific genes)
										from predominantly single-cell 'omics data through phylogenomic and
										machine learning approaches. This avenue of work is making it
										feasible to explore the evolution of genetic novelty, including the
										evolutionary pacing of lateral gene transfers, gene family birth/deaths,
										and expansion/contract over deep time scales. With an emphasis on lineages
										of eukaryotes where single-cell 'omics approaches dominate the
										available pools of data (e.g., ciliates and amoebozoans), there
										is ample opportunity to understand the evolutionary biology of genetic
										'novelty' across eukaryotes and its contributions to the on-going
										diversification of major eukaryotic lineages.
									<br><br>
									&nbsp &nbsp <strong>Relevant publications</strong>
									<br>
										&nbsp &nbsp &nbsp <a href="publication_pdfs/2024_MaurerAlcala_Ciliate_LSGF_PLoSOne.pdf">
											<strong>Maurer-Alcalá</strong> <i>et al.</i>, 2024</a><br>
										&nbsp &nbsp &nbsp <a href="publication_pdfs/2022_Cote_LHeureux_etal_LGT_PLoSGen.pdf">
									  	Cote-L'Heureux <i>et al.</i>, 2022</a>
									</p>


									<h3>Linking Genome Biology to Genome Evolution</h3>
									<p>When considering genome evolution, we often overlook the impacts of
										organismal biology and life history. Using ciliates, my work has focused
										on how their unusual genome biology (including developmental rearrangements and
										somatic hyperpolyploidy) manifest in their genome evolution.
										While ciliates represent extremes, their genome and developmental
										biology manifest in their evolution in interesting ways; extreme genome
										processing is tied to relaxed evolutionary pressures (Yan,
										<strong>Maurer-Alcalá</strong>, <i>et al.</i>, 2019) and may
										impact gene family birth/death too (<strong>Maurer-Alcalá</strong>,
										<i>et al.</i>, 2018, <strong>Maurer-Alcalá</strong>, <i>et al.</i>, 2024).
										This work has bridged diverse areas of biology (development, evolution,
										bioinformatics) and I am aiming to further explore these questions using
										experimental evolution and molecular biology techniques.
										<br><br>
										&nbsp &nbsp <strong>Relevant publications</strong>
										<br>
											&nbsp &nbsp &nbsp <a href="publication_pdfs/2024_MaurerAlcala_Ciliate_LSGF_PLoSOne.pdf">
												<strong>Maurer-Alcalá</strong> <i>et al.</i>, 2024.</a><br>
											&nbsp &nbsp	&nbsp <a href="publication_pdfs/2019_Yan_etal_GenArch_Evo_mBio.pdf">
												Yan, <strong>Maurer-Alcalá</strong> <i>et al.</i>, 2019</a><br>
											&nbsp &nbsp	&nbsp <a href="publication_pdfs/2018_MaurerAlcala_etal_SCG_Chilo_mBio.pdf">
												<strong>Maurer-Alcalá</strong> <i>et al.</i>, 2018</a><br>
											&nbsp &nbsp	&nbsp <a href="publication_pdfs/2016_MaurerAlcala_etal_NucArch_Chilo_GBE.pdf">
												<strong>Maurer-Alcalá</strong> and Katz, 2016</a><br>

									<h3>Developing Bioinformatic Approaches and Tools</h3>
									<p>Phylogenomics and comparative 'omics often use increasingly large datasets
										that present unique computational challenges. An important aspect
										of my work involves	the design and development of computational tools
										and approaches that improve access to those with less computational
										expertise. This includes the on-going development of <a href=https://github.com/xxmalcala/PhyG/wiki>PhyG</a>,
										a new modular phylogenomic pipeline incorporating natural language processing
										and machine learning approaches for robust phylogenomic analyses.
								 <p>Decontamination of eukaryotic dataset can be particularly challenging
										and often relies upon non-reproducible "manual curation". To improve
										curatorial reproducibility, I have developed <a href=https://github.com/xxmalcala/TIdeS>TIdeS</a>,
										a machine-learning framework for improved ORF prediction and decontamination
										of eukaryotic datasets (particularly transcriptomes), which provides a rapid
										and reproducible means to curate and prepare data for diverse analyses.
							  	<p>For some eukaryotes, namely ciliates, structural variations (SVs) between somatic
								 	  and germline genomes are widespread. To that end, <a href=https://github.com/xxmalcala/DSaGA>DSaGA</a>
									  provides a comparative genomic platform to summarize SVs, enabling comparisons
									  of transcriptomes or somatic genomes with complete or partial germlines.
									</p>
								</article>

						</div>
					</div>
				</section>

			<!-- Footer -->
				<div id="footer">

					<!-- Copyright -->
						<div class="copyright">
							<ul class="menu">
								<li>&copy; Xyrus X. Maurer-Alcalá</li><!--li>Design: <a href="http://html5up.net">HTML5 UP</a></li-->
							</ul>
						</div>

				</div>

		</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.dropotron.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
